src/components/select/multi-select/multi-select.component.tsx

```diff
@@ -529,6 +547,11 @@ export const MultiSelect = React.forwardRef<HTMLInputElement, MultiSelectProps>(
       isMouseDownReported.current = true;
     }
 
+    function handleOnListAction() {
+      setOpenState(false);
```

Comment:
Question: as the list closes when the button is clicked, will this really work for lazy loading?

@harpalsingh fyi